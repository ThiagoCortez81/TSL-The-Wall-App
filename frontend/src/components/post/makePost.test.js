import React from 'react';
import { shallow } from 'enzyme';
import MakePost from './makePost';
import TestUtils from "react-dom/test-utils";

describe('MakePost', () => {
    it('should render correctly in "debug" mode', () => {
        const component = shallow(<MakePost debug/>);

        expect(component).toMatchSnapshot();
    });

    it('shoud return success during making a post', () => {
        const component = shallow(<MakePost/>);

        component.instance().postToWall();
    });
});