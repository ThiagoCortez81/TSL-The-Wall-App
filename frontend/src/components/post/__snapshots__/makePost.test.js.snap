// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MakePost should render correctly in "debug" mode 1`] = `
<Row
  bsClass="row"
  className="mb-10 pr-30"
  componentClass="div"
>
  <Col
    bsClass="col"
    className="wall-post"
    componentClass="div"
    md={12}
    xs={12}
  >
    <div
      className="card"
    >
      <Col
        bsClass="col"
        componentClass="div"
        md={12}
        xs={12}
      >
        Make a post
        <FormControl
          bsClass="form-control"
          className="mb-10"
          componentClass="textarea"
          id="newPostContent"
          placeholder="What you want to share, null?"
        />
        <Button
          active={false}
          block={false}
          bsClass="btn"
          bsStyle="success"
          className="fw-6 float-right"
          disabled={false}
          onClick={[Function]}
        >
          Post to wall
        </Button>
      </Col>
    </div>
  </Col>
</Row>
`;
